### ISSUE TITLE
Switch between dark and light mode based on system settings

### ISSUE THREAD

**Issue by trosel** [lbryio/lbry-desktop#3017]
Since you use electron, I assume that this would be relevant: https://web.dev/prefers-color-scheme/

But there are also some libraries that can handles this for you as well https://github.com/Assortment/darkmodejs

**Comment by kauffj** [comment]
Great suggestion @trosel! Would you have any interest in trying to tackle this addition yourself?

**Comment by trosel** [comment]
@kauffj Yeah let me take a look at it

**Comment by trosel** [comment]
There are already some methods and flags for this behavior.

`onAutomaticDarkModeChange` and `automaticDarkModeEnabled`

I'm struggling to understand how this behavior currently works today, so I'm not comfortable with gutting this and replacing it with darkmodejs (although that is what I think should happen). 

See a react example here: https://github.com/Assortment/darkmodejs-demo/blob/master/src/pages/index.js

**Comment by kauffj** [comment]
@trosel rather than bringing in darkmode.js, I would look at how it is detecting the OS-level theme (it may be quite a succinct snipped) and then use that to set the initial value of `SETTINGS.THEME` in the [settings.js reduce](https://github.com/lbryio/lbry-desktop/blob/master/src/ui/redux/reducers/settings.js#L18).

If you do that, that's basically the entire ticket. 

(I suppose it'd be even better to have some sort of theme value that says "read at run-time" as a default, so that if the user changed their OS theming but never had changed the theme in the app, it would still change in the app, but this would be a great contribution without that.)

**Comment by trosel** [comment]
Yeah, could be. Honestly, I couldn't build the app and I'm not familiar with the code enough to do it blind.

**Comment by blokboy** [comment]
Is this issue still open? 

I think this could be a good starting point: https://github.com/Assortment/darkmodejs/blob/master/index.js

**Comment by kauffj** [comment]
@blokboy yes, it is still open and I suspect the code you linked would be the key to getting this done. It sounds like @trosel is not planning to work on this, but it'd be awesome if you did!

@trosel sorry to hear about the build issues. If you want to open an issue related to that, we'd be happy to look into those.

**Comment by seanyesmunt** [comment]
I believe electron@7 is required for this.

there is a new `nativeTheme` api called 
https://electronjs.org/blog/electron-7-0

**Comment by jessopb** [comment]
https://www.electronjs.org/docs/latest/tutorial/dark-mode
Now that we're on electron 15....
